www/docs/usage/commandline.md

```diff
@@ -76,8 +95,32 @@ tfswitch # Will install opentofu instead of terraform
 For example:
 
 ```bash
-export TF_ARCH=amd64
-tfswitch # Will install Terraform binary for amd64 architecture
+export TF_ARCH="amd64"
+tfswitch # Will install binary for amd64 architecture
+```
+
+### `TF_BINARY_PATH`
+
+`tfswitch` defaults to install to the `/usr/local/bin/` directory (and falls back to `$HOME/bin/` otherwise). The target filename is resolved automatically based on the `product` parameter.  
+`TF_BINARY_PATH` environment variable can be set to specify a **full installation path** (directory + file name). If target directory does not exist, `tfswitch` falls back to `$HOME/bin/` directory.
+
+For example:
+
+```bash
+export TF_BINARY_PATH="$HOME/bin/terraform" # Path to the file
+tfswitch # Will install binary as $HOME/bin/terraform
+```
+
+### `TF_INSTALL_PATH`
+
+`tfswitch` defaults to download binaries to the `$HOME/.terraform.versions/` directory.  
+`TF_INSTALL_PATH` environment variable can be set to specify a parent directory for `.terraform.versions` directory. Current user must have write permissions to the target directory. If the target directory does not exist, `tfswitch` will create it.
+
+For example:
+
+```bash
+export TF_INSTALL_PATH="/tmp" # Path to the directory where `.terraform.versions` directory resides
```

Comment:
I don't know if /tmp is a good idea - it sort of implies that all these are temporary files. But the cached binaries are relatively important (since it's expensive to keep downloading versions).
Maybe a dir in /var/cache would be a good example?